frontend/src/mocks/browser.ts

```diff
@@ -0,0 +1,10 @@
+import { http, HttpResponse } from 'msw'
```

Comment:
Put mocks directory outside of src directory

Reply:
Yeah should probably be in the tests directory. (i.e. tests/mocks)

Reply:
Fixed. 
Put it outside src but removed mocks directory